modules/nf-core/lofreq/somatic/main.nf

```diff
@@ -0,0 +1,42 @@
+process LOFREQ_SOMATIC {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::lofreq=2.1.5"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/lofreq:2.1.5--py38h588ecb2_4' :
+        'quay.io/biocontainers/lofreq:2.1.5--py38h588ecb2_4' }"
+
+    input:
+    tuple val(meta), path(tumor_bam), path(tumor_bai), path(normal_bam), path(normal_bai), path(target_bed)
+    path fasta
+    path fai
+
+    output:
+    tuple val(meta), path("*.vcf.gz"), emit: vcf
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def options_target_bed = target_bed ? "-l ${target_bed}" : ""
+    """
+    lofreq \\
+        somatic \\
+        --threads $task.cpus \\
+        $args \\
+        -f $fasta \\
+        -t $tumor_bam \\
+        -n $normal_bam \\
+        ${options_target_bed} \\
+        -o ${prefix}_lofreq_paired_
```

Comment:
```suggestion
        -o ${prefix}
```
This should be added in the `prefix` if you want these. This setup is the most flexible to the users of the module